File: C  programs/Problems on Recursion/Pattern Printing/Pattern2.c
/* Write a recursive program which display below pattern. 

Output : 1 2 3 4 5

*/

#include<stdio.h>

void DisplayI()
{
    int i=0;
	for(i=1;i<=5;i++)
	{
		printf("%d\n",i);
    }
}

void DisplayR()
{
	static int i=1;
	if(i<=5)
	{
		printf("%d\t",i);
		i++;
		DisplayR();
    }
}

int main()
{
  //DisplayI();
  DisplayR();
  return 0;
}

File: Java programs/Problems on Pattern Printing/Print Star pattern/Pattern2.java
/* Write a java program which Accept the number of rows and number of columns from user and Display below pattern.

Input:  iRow=4   iCol=4
Output : *  *  *  #
         *  *  #  *
         *  #  *  *
         #  *  *  *                         

*/
		  
import java.util.*;

class Pattern
{
    public void Pattern(int iRow,int iCol)
    {
	   int i=0;
	   int j=0;
	   for(i=1;i<=iRow;i++)
	   {  
          for(j=iCol;j>=1;j--)
		  {
			  if(i==j)
			  {
		         System.out.print("#\t");
	          }
			  else
			  {
				System.out.print("*\t");
              }
		  }			  
	     System.out.println();
       }   
	}   
}

class Lba401
{
    public static void main(String arg[])
    {
       Scanner sobj=new Scanner(System.in);

		int iValue1=0,iValue2=0;

        System.out.println("Enter number of Rows ");
        iValue1=sobj.nextInt();

		System.out.println("Enter number of Columns");
		iValue2=sobj.nextInt();

		Pattern pobj=new Pattern();
		
        pobj.Pattern(iValue1,iValue2);
	}	
}  	  
	  